### ISSUE TITLE
After save standalone: If you run EXE on other drives and other windows ... you will see an error message !!!

### ISSUE THREAD

**Issue by Shahin3d** [TheGameCreators/GameGuruRepo#347]
After save standalone: If you run EXE on other drives and other windows ... you will see an error message !!!

I made my own game - then I created a SETUP file - and I installed the game in the default drive.
Default drive address: C: \ Program Files (x86) or C: \ Program Files in Windows 10
The game was run with error ??? !!!
I saw an error while saving the game !!!
Eventually the game was broken and cut off !!!
I am very afraid !!!
View Error Images:
![erorr-0](https://user-images.githubusercontent.com/36647058/44284437-e2d12c80-a276-11e8-80fe-0fffa3e8b791.png)
![erorr-00](https://user-images.githubusercontent.com/36647058/44284451-f7adc000-a276-11e8-9022-7e07ff785a15.png)
![erorr-1](https://user-images.githubusercontent.com/36647058/44284666-c5509280-a277-11e8-8fd0-3bbeea1900dd.png)
![erorr-2](https://user-images.githubusercontent.com/36647058/44284673-cb467380-a277-11e8-9478-f829c052f88c.png)

I have researched and tested.
This is a problem when you copy or install the Game.exe folder in the following folders or in the default Windows drives.
C: \ Program Files (x86)
C: \ Program Files

Interesting point: When I installed or copy the game on other drives (other than the addresses listed): --- I did not see any problems in the run and was well implemented.

**Comment by Shahin3d** [comment]
I tested on Windows 7 and 8
Unfortunately, I saw other errors that had nothing to do with the default Windows drives!
After several times to save and load the game:
I received an error message
And the game was stopped.

![erorr](https://user-images.githubusercontent.com/36647058/44285297-0ba6f100-a27a-11e8-9510-4155963ca900.png)

**Comment by Shahin3d** [comment]
really why???
I work a lot of time and I've made the game ... eventually bad things happen !!!
And it makes me sad.
I want to sell my game but ... !!!

Please test yourself and see the result once.
For example, The Big Escape game
Of course on Windows 10
And in the drives listed above
Then on Windows 7 and 8

**Comment by Shahin3d** [comment]
Please fix this item faster
We should be able to run Game.exe on all windows and paths and drives.
..................... This is a great danger for us ...................... ......

I do not want you just type the bug tag for this problem!!!
And I can not wait for a few months
Please fix this item faster
Please fix this item faster
Please fix this item faster

**Comment by Teabone3** [comment]
You mention "install" the game rather than move/copy or place. You should be using a verified installer. One that uses proper certificate for Windows 10+. Which software are you using you have not specified here. You also mentioned you have created your own SETUP file?

**Comment by Shahin3d** [comment]
Yes . If you just copy the files to the default drive
You will see error messages when running the game
I mean copy the files from the folder (Documents \ Game Guru Files \ My Games) to C: \ Program Files (x86).
The operation that installer software performs to install the game.
This means installing in the default Windows drive: C: \ Program Files (x86) or C: \ Program Files
This is common for all software installs.
Of course, Windows 10, 7, and 8 are different types of errors.

If you test on Windows 10, you will understand what I mean.

Anyway:
I work with Advanced Installer 15.2 and I'm testing
I fixed my problem with Windows permissions for the exe file
I was able to make a good installation file
I have no problems with Windows 10 and I'm testing on Windows 7 and 8

Attachment image shows access permissions.
Of course, the test result in Windows 7 was good and there was no problem.

![permissions](https://user-images.githubusercontent.com/36647058/44448490-a9931680-a600-11e8-9e5a-d043392cc92e.png)


However:
Maybe you can find a better solution or access additional Windows permissions for GG manager.

**Comment by Shahin3d7** [comment]
This is the correct access:
![untitled](https://user-images.githubusercontent.com/43903231/46697000-793f2000-cc20-11e8-94f4-62aaf5a5e2aa.png)

**Comment by Shahin3d7** [comment]
I used the (Advanced Installer) software.
I built my own installation file.
It worked very well on all windows(10 & 8 & 7).
(Advanced Installersoftware): It is an advanced software and licensed to all Windows.
Just select the permissions for your installation file.
Of course I created the file for my game.
(Help image above)
I tested it on all windows and drives.
The result was positive and I did not see any problems.
